src/js/pages/jobs/JobDetailPage.js

```diff
@@ -10,6 +10,8 @@ import mixin from "reactjs-mixin";
 import JobFormModalContainer from "#PLUGINS/jobs/src/js/JobFormModalContainer";
 import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
 import JobsBreadcrumbs from "#PLUGINS/jobs/src/js/components/JobsBreadcrumbs";
+import jobsMenu from "#PLUGINS/jobs/src/js/jobsMenu";
```

Comment:
❓ why is this starting with a small `j` at all? 🤔

Reply:
please look at the jobsDelete answer.